### PR TITLE
[Notifier] Adds documentation for Slack stuff added in 5.3

### LINKED ISSUE #15315: [Notifier] Add SlackOptions::threadTs() to send message as reply
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#40992
| PR author(s) | @WaylandAce
| Merged in    | 5.3-dev

### PR DESCRIPTION
**Pull request by 0x346e3730**
Closes #15315,  #14694 &  #14653 by adding documentation for these additions related to the notifier.

### CODE REVIEW

**Comment by 0x346e3730** [comment]
I don't understand why the checks are failing, the classes exist in Symfony 5.3...

**Comment by javiereguiluz** [comment]
Antonin thanks for this great contribution that closes so many pending issues 👏

File: notifier/chatters.rst [hunk 1]
```diff
@@ -135,6 +135,120 @@ The result will be something like:
...
+    $chatter->send($chatMessage);
+
+The result will be something like:
+
+.. image:: /_images/notifier/slack/slack-header.png
+   :align: center
+
+.. versionadded:: 5.3
+
+    The `SlackHeaderBlock` class was introduced in Symfony 5.3.
```
Comment (OskarStark) [inline notifier/chatters.rst:176]: Double backticks needed

Comment (javiereguiluz) [reply notifier/chatters.rst:176]: Yes, I fixed several of these single back tick occurrences while merging. Thanks.

Comment (OskarStark) [reply notifier/chatters.rst:176]: Thank you

Comment (0x346e3730) [reply notifier/chatters.rst:176]: Thank you Javier, and thank you Oskar for letting me know I'll double check in the future 👍

Comment (OskarStark) [reply notifier/chatters.rst:176]: Thanks for contributing to the docs 😎👍🏻

**Pull request merged**